lib/utils.js

```diff
@@ -33,7 +33,8 @@ function debrack (s) {
 }
 
 function uriToFilename (uri, base) {
-  var filename = path.join(base, uri)
+  var decoded = uri.split('/').map(decodeURIComponent).join('/')
```

Comment:
I think we're going to run into problems with this approach. What if the file (or container) was uploaded as already uri-encoded? Say, for example, it was uploaded as `hello%20world.ttl`? And exists on disk as `hello%20world.ttl`. If you then issue a GET on it and this method `uriToFilename()` is invoked, it will try and look for the filename `hello world.ttl` and not find it.